Fun Commands/Mini-Games/Dyno-RPG/README.md

```diff
@@ -50,11 +51,33 @@ As well as the following channels:
 - `sacrificial-offerings`
 - `divine-council`
 
-It is suggested that the `divine-council` channel be useable ONLY by serverMods as it is the channel required for using the Divine Council commands.
+
+### How to
+A simple guide for setting up roles and channels for the commands to work as intended
 
 ***The `Adventurer` role is the base role for playing Dyno RPG. As such it is suggested to use the role selection menu when adding thse commands to make the commands require it.***
 
+***Deathmatch Commands should be allowed only in `the-deathpits`***
+
+***Divine Council Commands should be limited to serverMod only and only allowed to be used in `divine-council`, while the response can preferaly be set to world-gates` if desired***
+
+***Plaugue and Fester should only be useable if the person calling them is `Infected`***
+
+***Divine Council is not actually a role, it's a nickname for serverMods. as such I decided not to make a seperate role for it.***
+
+***The `rpg-cmds` command should be accessible to `Adveturer` roles only, while the `register` command should be acessible to everyone in th server so people can join Avelon***
+
+***All commands should preferrably also be set to both silent and delete so as to prevent anyone from knowing who is doing what***
```

Comment:
Spelling
```suggestion
***All commands should preferably also be set to both silent and delete so as to prevent anyone from knowing who is doing what***
```